### ISSUE TITLE
Expand out the functionality of the contact us page

### ISSUE THREAD

**Issue by wasade** [qiime/qiime_web_app#616]
Including a drop down menu for the type of question can help differentiate the subject used on the email. In addition, we need to properly setup the reply-to address field so that it goes to the participant, and add on some type of differentiation in the subject line so gmail doesn't aggregate everything.

**Comment by adamrp** [comment]
Has gmail been aggregating emails?  I haven't noticed this.

**Comment by wasade** [comment]
Remember the international incident?

**Comment by adamrp** [comment]
That was a result of Jeff sending an email to all participants with our
email address as the reply to address (this was a nightmare, but not caused
by our help request functionality).  We can, of course, bulletproof things
by appending a GUID to the end of every email sent through our interface...
not sure if it's necessary though, since it's never been a problem.

**Comment by wasade** [comment]
Nope, that was sent by me by the help account. We do have some emails that
aggregate if the subject is the same on the help account though, but I'm
not sure of the exact situations when it does

**Comment by wasade** [comment]
...unless I'm misremembering, been a bit of an insane few months I suppose

**Comment by adamrp** [comment]
Oh, I'm not sure... it's the email on the account with the subject "To the
international participants of the American Gut Project." that has 92
replies (it's not signed with anyone's particular name). It looks like it
might have been sent directly from the help account, but still not through
the interface on the website.

**Comment by wasade** [comment]
Definitely not through the website. Concern is just what gmail is going to
do

**Comment by teravest** [comment]
We could let the participants write the subject, or use the kit id and have
the page put the question from the drop down in the first line of the
email, then filter based on email body starts with in gmail.

**Comment by adamrp** [comment]
I honestly have never seen gmail aggregate emails from different senders
when they use the help interface.  If we are truly worried about this, then
using a GUID for each email sent from the help interface is the way to go.
 I don't have strong feelings about letting the user write the subject
line, but in any case I think we should inject a signal into the subject
line (based on the question selected from the dropdown menu), and use a
gmail filter to apply labels to incoming messages according to these
signals by having one label per dropdown menu option.  This will facilitate
distribution of messages to the individuals most equipped to answer them.

**Comment by ElDeveloper** [comment]
:+1:  I like the idea of adding a tag to the subject such that we apply some gmail magic and organize these neatly.
